FreydCategoriesForCAP/gap/AdditiveClosure.gi

```diff
@@ -1356,6 +1362,256 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_ADDITIVE_CLOSURE,
         
     fi;
     
+    if HasIsMonoidalCategory( underlying_category ) and IsMonoidalCategory( underlying_category ) then
+        
+        ##
+        AddTensorUnit( category,
+          function( cat )
+            
+            return ObjectConstructor( cat, [ TensorUnit ( UnderlyingCategory( cat ) ) ] );
```

Comment:
```suggestion
            return ObjectConstructor( cat, [ TensorUnit( UnderlyingCategory( cat ) ) ] );
```

Reply:
Ah, I remember seeing this one, but missed asking @damian-delafuente to change it. Thanks :)

Reply:
done